xml-parsing/src/main/java/org/sonarsource/analyzer/commons/xml/checks/SonarXmlCheckVerifier.java

```diff
@@ -76,8 +75,7 @@ public static void verifyNoIssue(String relativePath, SonarXmlCheck check) {
   private static SonarXmlCheckVerifier createVerifier(String fileName, SonarXmlCheck check) {
     File file = new File(new File(BASE_DIR.toFile(), check.getClass().getSimpleName()), fileName);
 
-    SensorContextTester context = SensorContextTester.create(BASE_DIR)
-      .setActiveRules(new ActiveRulesBuilder().create(RULE_KEY).activate().build());
+    SensorContextTester context = SensorContextTester.create(BASE_DIR);
```

Comment:
why we don't need the rule to be activated anymore?

Reply:
@saberduck good point, no idea. Probably SensorContextTester accepts issues even their rules are not activated. I think we didn't even check that when adding this line initially.